src/rust/plugin-registry/src/client.rs

```diff
@@ -33,6 +39,33 @@ pub struct PluginRegistryServiceClient<T> {
     inner: _PluginRegistryServiceClient<T>,
 }
 
+const HOST_ENV_VAR: &'static str = "GRAPL_PLUGIN_REGISTRY_HOST";
+const PORT_ENV_VAR: &'static str = "GRAPL_PLUGIN_REGISTRY_PORT";
+
+impl PluginRegistryServiceClient<Channel> {
```

Comment:
this is basically the same code as model-plugin-deployer/src/client.rs - our other extant rust grpc service